src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -250,7 +250,9 @@ def convert_pytorch_sharded_state_dict_to_flax(shard_filenames, flax_model):
     for shard_file in shard_filenames:
         # load using msgpack utils
         pt_state_dict = torch.load(shard_file)
-        pt_state_dict = {k: v.numpy() for k, v in pt_state_dict.items()}
+        pt_state_dict = {
+            k: v.numpy() if v.dtype != torch.bfloat16 else v.float().numpy() for k, v in pt_state_dict.items()
+        }
```

Comment:
cc @sanchit-gandhi could you validate if this is alright?

Reply:
Hi @sanchit-gandhi,
This change would be required to do 
```
from transformers import FlaxMistralForCausalLM
model = FlaxMistralForCausalLM.from_pretrained("mistralai/Mistral-7B-v0.1", from_pt=True)
```
If it is to be excluded it can still be loaded with
```
from transformers import FlaxMistralForCausalLM
model = FlaxMistralForCausalLM.from_pretrained("mistralai/Mistral-7B-v0.1", revision='refs/pr/95')
```
as per PR https://huggingface.co/mistralai/Mistral-7B-v0.1/discussions/95

Reply:
Updated weight conversion is spot-on! Marking as resolved ✅